src/components/TextField/TextFieldConstants.ts

```diff
@@ -28,7 +28,8 @@ export enum TextFieldTextType {
   SEARCH = "search",
   DATE = "date",
   DATE_TIME = "datetime-local",
-  NUMBER = "number"
+  NUMBER = "number",
+  EMAIL = "email"
```

Comment:
Please lets separate it to a different PR 🙏🏼

Reply:
Lets remove it for now

Reply:
We need this for another question. I can create another PR but we need this